pkg/kubectl/resource/builder.go

```diff
@@ -184,6 +184,15 @@ func (b *Builder) ResourceTypes(types ...string) *Builder {
 	return b
 }
 
+// ResourceNames accepts a default type and one or more names, and creates tuples of
+// resources
+func (b *Builder) ResourceNames(resource string, names ...string) *Builder {
+	for _, name := range names {
```

Comment:
Ok, I see that this has to be the equivalent of `ResourceTypeOrNameArgs` for names